### PR TITLE
FAQ eu-dcc-check

### PR DESCRIPTION
**Pull request by dsarkar**
**DE**
image

**EN**

image

---
Internal Tracking ID: [EXPOSUREAPP-9373](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9373)

### CODE REVIEW

File: src/data/faq_de.json [hunk 1]
```diff
@@ -214,6 +214,16 @@
...
+                        "anchor": "eu_dcc_check",
+                        "active": true,
+                        "textblock": [
+                            "In Deutschland können Dritte nur mit der CovPassCheck-App verlässlich überprüfen, ob es sich um ein valides Impf-, Genesenen-, oder Testzertifikat handelt. Zusätzlich sollte eine Identitätsprüfung mit Lichtbildausweis stattfinden. Bitte beachten Sie, dass in anderen Ländern gegebenenfalls andere Apps zur Zertifikatsprüfung verwendet werden.",
+                            "Eine reine Sichtprüfung der Zertifikate reicht nicht aus, da beispielsweise manipulierte Screenshots, Apps oder Systemeinstellungen (z.&nbsp;B. geändertes Datum) verwendet werden könnten. Die CovpassCheck-App ist eine sichere Lösung für z.B. Gewerbetreibende und Behörden, mit der digitale COVID-Zertifikate der EU zuverlässig geprüft werden können. Weitere Informationen und Download-Links zur CovPassCheck-App finden Sie hier: <a href='https://www.digitaler-impfnachweis-app.de/covpasscheck-app/' target='_blank' rel='noopener noreferrer'>COVID-Zertifikate der EU direkt per App prüfen</a>.",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:224]: The second z.B. should also be written with `&nbsp;`

File: src/data/faq.json [hunk 2]
```diff
@@ -214,6 +214,16 @@
...
+                            "A pure visual check of the certificates is not sufficient, as manipulated screenshots, apps or system settings (e.g. changed date) could be used, for example. The CovPassCheck-App is a secure solution for e.g. traders and public authorities to reliably check EU digital COVID certificates. More information and download links to the CovPassCheck app can be found here: <a href='https://www.digitaler-impfnachweis-app.de/en/covpasscheck-app/l' target='_blank' rel='noopener noreferrer'>Check EU COVID certificates directly via app</a>. Please note that third parties in other countries may use other apps to verify certificates.",
+                            "You can check certificates for validity on your own directly in the Corona-Warn-App without the need of using the CovPassCheck-App. For more information, see the blog <a href='../blog/2021-07-28-cwa-version-2-6/' target='_blank' rel='noopener noreferrer'>Users can add local 7-day incidences and check certificates for validity before traveling</a> and the FAQ article <a href='#cert_eu_travel' target='_blank' rel='noopener noreferrer'>Is my certificate valid in the country I am travelling to</a>?"
```
Comment (MikeMcC399) [inline src/data/faq.json:225]: `</a>?` question mark at end of this line should be inside hyperlink `?</a>`

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Minor typo comments only.

File: src/data/faq.json (hunk 2 above)
Comment (dsarkar) [reply src/data/faq.json]: ok, thanks

File: src/data/faq_de.json (hunk 1 above)
Comment (dsarkar) [reply src/data/faq_de.json]: right, thanks

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for the review

**Pull request merged**